docs/installation.md

```diff
@@ -74,7 +74,7 @@ import * as zarr from "https://cdn.skypack.dev/zarr";
 
 ```html
 <!-- Import as UMD in HTML -->
-<script src="https://unpkg.com/zarr/dist/zarr.umd.js"></script>
+<script src="https://unpkg.com/zarr/zarr.umd.js"></script>
```

Comment:
@gzuidhof @oeway I reverted the change in #74 because the build as of #63 lifts the contents of `dist/` to the top-level of the tarball. See: https://unpkg.com/zarr@0.4.0/zarr.umd.js

Reply:
Sorry I didn't comment on the last PR sooner.